### DISCUSSION TITLE
how to add environment to remix ide?

### DISCUSSION THREAD

**Discussion by a-rya13** [Cyfrin/foundry-full-course-cu#2497] [Q&A]
how to add javascript environment to remix ide as it is not available??
![Uploading Screenshot 2024-09-17 005357.png…]()

**Comment by EngrPips** [comment]
Hello @a-rya13, The image you sent is not loading, and I am unsure what you mean by `how to add javascript environment to remix ide `

**Reply by a-rya13** [reply]
![Screenshot 2024-09-17 164222](https://github.com/user-attachments/assets/50a87501-87b8-4e9a-a935-c1c1b7549d39)
here in the list of environment I also want  to add javascript to test my code

**Reply by EngrPips** [reply]
I don't know how to do that. Let's hope someone who knows how to do it sees this discussion soon and helps.
